compiler/noirc_frontend/src/hir/comptime/interpreter/builtin.rs

```diff
@@ -232,8 +232,11 @@ impl<'local, 'context> Interpreter<'local, 'context> {
             "unresolved_type_is_field" => unresolved_type_is_field(interner, arguments, location),
             "unresolved_type_is_unit" => unresolved_type_is_unit(interner, arguments, location),
             "zeroed" => zeroed(return_type, location.span),
+            blackbox if BlackBoxFunc::is_valid_black_box_func_name(blackbox) => {
+                self.call_foreign(blackbox, arguments, location)
+            }
```

Comment:
For other reviewers, we discussed this and agreed that this is suboptimal but there's a number of blackbox functions in the stdlib which are marked as builtins and we didn't want to make those changes to the stdlib in this PR. This can be cleaned up in a followup.

Reply:
Created https://github.com/noir-lang/noir/issues/6594